### ISSUE TITLE
Non recoverable error in on_player_setup_blueprint

### ISSUE THREAD

**Issue by nivertius** [Sopel97/railway_signalling_overseer#5]
Version: 1.1.19

Stack trace:
```
Error while running event railway_signalling_overseer::on_player_setup_blueprint (ID 70)
__railway_signalling_overseer__/control.lua:557: attempt to index local 'entity' (a nil value)
stack traceback:
	__railway_signalling_overseer__/control.lua:557: in function 'is_signal_marked_correct'
	__railway_signalling_overseer__/control.lua:1847: in function <__railway_signalling_overseer__/control.lua:1834>
```

Happens when selecting train stop for copying. The selection includes entities from [Bulk Rail Loader](https://mods.factorio.com/mod/railloader). [Save file](https://ufile.io/2ljyu1y3), to trigger the problem please select "Main" train stop.

**Comment by Sopel97** [comment]
Thanks for the report. I'll take a look tomorrow.

**Comment by Sopel97** [comment]
Fixed in 1.1.20

**Comment by nivertius** [comment]
I can confirm that in 1.1.20 this issue is not present. Thank you!
